src/pageServices/contentScripts/highlight.js

```diff
@@ -128,10 +128,17 @@ export const highlightOnPage = () => {
     div.querySelector(".jdn-class").textContent = element.name;
   };
 
+  const updateTooltipXpath = (element) => {
+    if (element.locator.taskStatus === "SUCCESS" && tooltip.getAttribute("jdn-element-id") === element.element_id) {
```

Comment:
you can import `LocatorTaskStatus` from `src\features\locators\types\locator.types.ts` and compare with it, instead of "SUCCESS" string